File: Practica 1/Punto.py
import math
class Punto:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        return f"({self.x}, {self.y})"

    def coord_polares(self):
        r = math.sqrt(self.x**2 + self.y**2)
        a = math.atan(self.y / self.x)
        return f"(r: {r}, θ: {a})"

    def __str__(self):
        return f"Punto: {self.coord_cartesianas()}"

p = Punto(3, 4)
print(p)
print("Coordenadas polares:", p.coord_polares())

File: Practica 1/Punto.java
package tareaCodernada;
import java.lang.Math;
public class Punto {
    public float x;
    public float y;
    
    public Punto(float x,float y){
        this.x=x;
        this.y=y;
    }
    public String coord_cartesianas(){
        return "("+this.x+","+this.y+")";
    }
    public void coord_polares(){
        float r=(float) Math.sqrt((x*x)+(y*y));
        float v=y/x;
        float a=(float) Math.atan(v);
        System.out.println("Radio: "+r+" Angulo: "+a);
    }
    public String toString(){
        return "Punto"+coord_cartesianas();
    }
}
